core/scale/scale_decoder_stream.hpp

```diff
@@ -16,6 +16,9 @@
 namespace kagome::scale {
   class ScaleDecoderStream {
    public:
+    // special tag to differentiate decoding streams from others
+    static constexpr std::true_type is_decoder_stream;
```

Comment:
```suggestion
    using is_decoder_stream = std::true_type;
```

Reply:
does not work

Reply:
what does it say?

Reply:
https://gist.github.com/kamilsa/e45f1bbaf7cc3950e0a0e12f46a58bf6

Reply:
Okay, constexpr static is good